README.md

```diff
@@ -8,3 +8,13 @@ To build and run this application please use the `kraft` script:
 
 For more information about `kraft` type ```kraft -h``` or read the
 [documentation](http://docs.unikraft.org).
+# Dependencies
+To run duktape, a root filesystem must be mounted.
+To build, python 2.7 needs to be installed under the "python" name with pyYaml make sure you do not have a "python" to "python3". This can be done with the following bash commands:
+
+    sudo apt install python2
+    sudo ln -s /bin/python2.7 /usr/bin/python
```

Comment:
Don't do these, as it may broke other people dependencies.

Reply:
Instead you can update [here](https://github.com/unikraft/lib-duktape/blob/staging/Makefile.uk#L82)  to use `python2` instead of `python`. Another PR would be needed.